package.json

```diff
@@ -43,7 +43,7 @@
     "request": "^2.87.0",
     "request-capture-har": "^1.2.2",
     "rimraf": "^2.5.0",
-    "semver": "^5.1.0",
+    "semver": "atlassian-forks/node-semver.git#refs/heads/alwasIncludePrerelease",
```

Comment:
Is that right that the tag name is misspelt? `alwas` not `always`?

Reply:
Oops, yes: https://github.com/atlassian-forks/node-semver/tree/alwasIncludePrerelease